src/services/BiometricAuth.js

```diff
@@ -14,9 +14,10 @@ import { BlurView } from 'expo-blur';
 import { _, BiometricAuth as t } from '../utils/i18n';
 import colors from '../styles/colors';
 import BlockTemplate from '../components/BlockTemplate';
+import { Config, PayUTC } from '../redux/actions';
 
-export const defaultSecurity = ['transfer', 'refill', 'badge-locking'];
-export const advancedSecurity = ['transfer', 'refill', 'badge-locking', 'app-opening'];
+export const defaultSecurity = ['TRANSFER', 'REFILL', 'BADGE_LOCKING'];
```

Comment:
Actually, you should have done that:
```js
export const TRANFER = 'TRANSFER';
export const REFILL = 'REFILL';
export const BADGE_LOCKING = 'BADGE_LOCKING';
export const APP_OPENING = 'APP_OPENING';

export const DEFAULT_SECURITY = [TRANSFER, REFILL, BADGE_LOCKING];
export const ADVANCED_SECURITY = [TRANSFER, REFILL, BADGE_LOCKING, APP_OPENING];
```

and then, you import the consts in your componenets